cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/query/QueryExecutor.java

```diff
@@ -55,7 +55,7 @@
 
     /**
      *  Constructs a new QueryExecutor using the indexes in 'database' to find documents from
-     *  'datastore'.
+     *  'DocumentStore'.
```

Comment:
Comment probably needs to be reworked - quotes imply that these are parameter names.